docs/backup-restore.md

```diff
@@ -1,134 +1,204 @@
+
+In addition, you should pause for a sufficient amount of time (for example, 10 seconds) and use the mid-time of the paused duration as a restore point since clock synchronization cannot perfectly synchronize clocks between nodes.
+{% endcapture %}
 
-#### SQLite (backup)
+<div class="notice--info">{{ notice--info | markdownify }}</div>
 
-Use the [`.backup` command](https://www.sqlite.org/cli.html#special_commands_to_sqlite3_dot_commands_) along with `.timeout`.
+How you create a transactionally consistent backup depends on the type of database that you're using. Select your database to see how to create a transactionally consistent backup for ScalarDB.
 
-See also [this post](https://stackoverflow.com/questions/23164445/bash-sqlite3-backup-command), for example.
+<div id="tabset-2">
+<div class="tab">
+  <button class="tablinks" onclick="openTab(event, 'Cassandra2', 'tabset-2')" id="defaultOpen-2">Cassandra</button>
+  <button class="tablinks" onclick="openTab(event, 'Cosmos_DB_for_NoSQL2', 'tabset-2')">Cosmos DB for NoSQL</button>
+  <button class="tablinks" onclick="openTab(event, 'DynamoDB2', 'tabset-2')">DynamoDB</button>
+</div>
 
-#### Amazon RDS or Azure Database for MySQL/PostgreSQL (backup)
+<div id="Cassandra2" class="tabcontent" markdown="1">
 
-You can restore to any point within the backup retention period with the automated backup feature.
+Cassandra has a built-in replication feature, so you do not always have to create a transactionally consistent backup. For example, if the replication factor is set to `3` and only the data of one of the nodes in a Cassandra cluster is lost, you won't need a transactionally consistent backup because the node can be recovered by using a normal, transactionally inconsistent snapshot and the repair feature.
 
-### Backup with explicit pausing
+However, if the quorum of cluster nodes loses their data, you will need a transactionally consistent backup to restore the cluster to a certain transactionally consistent point.
 
-One way to create a transactionally-consistent backup is to take a backup while ScalarDB cluster does not have outstanding transactions.
-If an underlying database supports a point-in-time snapshot/backup mechanism, you can take a snapshot during the period.
-If an underlying database supports a point-in-time restore/recovery mechanism, you can set a restore point to a time (preferably the mid-time) in the period.
+To create a transactionally consistent cluster-wide backup, pause the application that is using ScalarDB or ScalarDB Server and take snapshots of the nodes as described in [Back up with explicit pausing](#back-up-with-explicit-pausing) or stop the Cassandra cluster, take copies of all the data in the nodes, and start the cluster.
```

Comment:
For consistency in how we seem to use `backup` more often than snapshot, can we change this sentence as follows?:

```suggestion
To create a transactionally consistent cluster-wide backup, pause the application that is using ScalarDB or ScalarDB Server and create backups of the nodes as described in [Back up with explicit pausing](#back-up-with-explicit-pausing) or stop the Cassandra cluster, take copies of all the data in the nodes, and start the cluster.
```